views/templates/admin/configuration.tpl

```diff
@@ -1,119 +1,156 @@
+      <p for="gsitemap_meta">{l s='Indicate the pages that you do not want to include in your Sitemaps file:' d='Modules.Gsitemap.Admin'}</p>
+      <ul>
+         {foreach from=$store_metas item=store_meta}
+         <li style="float: left; width: 400px; margin-bottom: 15px">
+            <input type="checkbox" class="gsitemap_metas" name="gsitemap_meta[]"{if in_array($store_meta.id_meta, $gsitemap_disable_metas)} checked="checked"{/if} value="{$store_meta.id_meta|intval}" /> {$store_meta.title|escape:'htmlall':'UTF-8'} [{$store_meta.page|escape:'htmlall':'UTF-8'}]
+         </li>
+         {/foreach}
+      </ul>
+      <br>
+      <div class="margin-form" style="clear: both;">
+         <br>
+         <button type="submit" class="btn btn-primary btn-lg" name="SubmitGsitemap" onclick="$('#gsitemap_loader').show();">{l s='Generate Sitemap' d='Modules.Gsitemap.Admin'}</button>
+         <br><br>
+         <div class="alert alert-info" role="alert">
+            {l s='Generating a sitemap can take several minutes' d='Modules.Gsitemap.Admin'}</p>
+         </div>
+      </div>
+      <p id="gsitemap_loader" style="text-align: center; display: none;"><img src="../img/loader.gif" alt=""/></p>
+   </form>
+</div>
+
+<div class="panel">
+   <h3><i class="icon icon-tags"></i> {l s='Informations' d='Modules.Gsitemap.Admin'}</h3>
+   <p>
+      <strong>{l s='You have two ways to generate Sitemap:' d='Modules.Gsitemap.Admin'}</strong><br><br>
+      1. <strong>{l s='Manually:' d='Modules.Gsitemap.Admin'}</strong> {l s='using the form above (as often as needed)' d='Modules.Gsitemap.Admin'}<br>
+      <br><span style="font-style: italic;">{l s='-or-' d='Modules.Gsitemap.Admin'}</span><br><br>
+      2. <strong>{l s='Automatically:' d='Modules.Gsitemap.Admin'}</strong> {l s='Ask your hosting provider to setup a "Cron task" to load the following URL at the time you would like:' d='Modules.Gsitemap.Admin'}
+      <a href="{$gsitemap_cron|escape:'htmlall':'UTF-8'}" target="_blank">{$gsitemap_cron|escape:'htmlall':'UTF-8'}</a><br><br>
+      {l s='It will automatically generate your XML Sitemaps.' d='Modules.Gsitemap.Admin'}
+   </p>
+   {/if}
+   </p>
+</div>
+
 <script type="text/javascript">
-$(document).ready(function() {
-	
-	if ($('.gsitemap_metas:checked').length == $('.gsitemap_metas').length)
-		$('.check').parent('label').children('span').html("{l s='uncheck all' mod='gsitemap'}");
-	
-	
-	$('.check').toggle(function() {
-		$('.gsitemap_metas').attr('checked', 'checked');
-		$(this).parent('label').children('span').html("{l s='uncheck all' mod='gsitemap'}");
-	}, function() {
-		$('.gsitemap_metas').removeAttr('checked');
-		$(this).parent('label').children('span').html("{l s='check all' mod='gsitemap'}");
-	});
-});
+   $(document).ready(function() {
+   	
+   	if ($('.gsitemap_metas:checked').length == $('.gsitemap_metas').length)
+   		$('.check').parent('label').children('span').html("{l s='uncheck all' d='Modules.Gsitemap.Admin'}");

```

Comment:
Can you remove tabs in this file please?

Reply:
Fixed